{ 'fall_position_back': { 'head': [ { 'duration': 1.0,
                                      'name': u'fall_position_back',
                                      'positions': [0.0, 0.0012],
                                      'starttime': 0.0}],
                          'left_arm': [ { 'duration': 1.0,
                                          'name': u'fall_position_back',
                                          'positions': [ -2.2981,
                                                         1.0113,
                                                         0.0152,
                                                         -0.0308,
                                                         -0.8378,
                                                         0.0054,
                                                         -0.0472],
                                          'starttime': 0.0}],
                          'left_leg': [ { 'duration': 1.0,
                                          'name': u'fall_position_back',
                                          'positions': [ -0.0027,
                                                         -0.0948,
                                                         -0.174,
                                                         0.0544,
                                                         -0.096,
                                                         -0.0783],
                                          'starttime': 0.0}],
                          'right_arm': [ { 'duration': 1.0,
                                           'name': u'fall_position_back',
                                           'positions': [ 2.2981,
                                                          -1.0113,
                                                          -0.0152,
                                                          0.0308,
                                                          0.8378,
                                                          -0.0054,
                                                          0.0472],
                                           'starttime': 0.0}],
                          'right_leg': [ { 'duration': 1.0,
                                           'name': u'fall_position_back',
                                           'positions': [ 0.0027,
                                                          0.0948,
                                                          0.174,
                                                          -0.0544,
                                                          0.096,
                                                          0.0783],
                                           'starttime': 0.0}],
                          'torso': [ { 'duration': 1.0,
                                       'name': u'fall_position_back',
                                       'positions': [0.0087, 0.697],
                                       'starttime': 0.0}]}}
